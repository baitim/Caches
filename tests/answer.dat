file			lfu_hits	belady_hits		ratio
test_01.dat		703			5294			7,53
test_02.dat		918			6321			6,89
test_03.dat		435			4273			9,82
test_04.dat		337			4055			12,03
test_05.dat		489			4625			9,46
test_06.dat		644			4528			7,03
test_07.dat		578			4090			7,08
test_08.dat		256			2826			11,04
test_09.dat		507			4008			7,91
test_10.dat		492			3009			6,12
test_11.dat		315			3435			10,90
test_12.dat		407			2980			7,32
test_13.dat		756			5276			6,98
test_14.dat		376			2748			7,31
test_15.dat		364			3996			10,98
test_16.dat		193			2033			10,53
test_17.dat		879			6242			7,10
test_18.dat		850			5326			6,27
test_19.dat		164			1930			11,77
test_20.dat		344			3884			11,29
