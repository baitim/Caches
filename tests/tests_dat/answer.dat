file			lfu_hits	belady_hits		ratio
tests_dat/test_01.dat		2117			5639			2,66
tests_dat/test_02.dat		1655			5524			3,34
tests_dat/test_03.dat		2884			7724			2,68
tests_dat/test_04.dat		1811			6550			3,62
tests_dat/test_05.dat		1574			4168			2,65
tests_dat/test_06.dat		1982			4586			2,31
tests_dat/test_07.dat		1775			5953			3,35
tests_dat/test_08.dat		2174			5540			2,55
tests_dat/test_09.dat		1092			3666			3,36
tests_dat/test_10.dat		1994			6596			3,31
