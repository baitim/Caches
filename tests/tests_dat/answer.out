file			lfu_hits	arc			belady_hits		ratio21	ratio31	ratio32
test_001.in		2333		2269		5553			0.97	2.38	2.45
test_002.in		1868		1854		5119			0.99	2.74	2.76
test_003.in		1404		1332		3742			0.95	2.67	2.81
test_004.in		1530		1607		4416			1.05	2.89	2.75
test_005.in		1495		1443		5216			0.97	3.49	3.61
test_006.in		2703		2739		8014			1.01	2.96	2.93
test_007.in		1276		1266		3981			0.99	3.12	3.14
test_008.in		2486		2440		7039			0.98	2.83	2.88
test_009.in		3128		3036		8442			0.97	2.7 	2.78
test_010.in		2270		2262		5155			1.0 	2.27	2.28
